Progress Bar Addition

Chuk116 opened the pull request:
**changes**
Adds a progressbar as shown in the figma mockup. Slight tweak to the design for the mobile view for now due to mobile compatibility reasons. A fix to make it look like as shown on the mockup may be in the works if necessary.


**screenshots**
*Laptop*
<img width="1514" alt="entry-progress-bar-laptop" src="https://user-images.githubusercontent.com/28410610/104247736-31219000-541d-11eb-9ded-886c9f93f0ee.png">

*iPad Pro*
<img width="324" alt="entry-progress-bar-ipadpro" src="https://user-images.githubusercontent.com/28410610/104247771-3aaaf800-541d-11eb-9efd-b5b468903b95.png">

*iPad*
<img width="317" alt="entry-progress-bar-ipad" src="https://user-images.githubusercontent.com/28410610/104247965-865da180-541d-11eb-96a5-1e38d735e848.png">

*Mobile*
<img width="209" alt="entry-progress-bar-mobile" src="https://user-images.githubusercontent.com/28410610/104248342-29aeb680-541e-11eb-9716-930530d7e142.png">

okusio commented:
yessss!!!! 
![Screen Shot 2021-01-11 at 6 12 46 PM](https://user-images.githubusercontent.com/57499412/104249040-a484cb00-5438-11eb-89e9-e814fb3189dc.png)
Can we make the line skinnier (half the size it currently is). + take away the shadow on the circle. I love love love the checkmark!!!!!
Also, the line disappears on mobile.

Chuk116 commented:
> yessss!!!!
> ![Screen Shot 2021-01-11 at 6 12 46 PM](https://user-images.githubusercontent.com/57499412/104249040-a484cb00-5438-11eb-89e9-e814fb3189dc.png)
> Can we make the line skinnier (half the size it currently is). + take away the shadow on the circle. I love love love the checkmark!!!!!
> Also, the line disappears on mobile.

I can make the line skinnier. 
<img width="113" alt="Screen Shot 2021-01-11 at 3 32 34 PM" src="https://user-images.githubusercontent.com/28410610/104250544-a8a5ee00-5422-11eb-9645-6ef6f81e8d80.png">


The reason I added the shadow is that it serves as an extra subtle indicator for which step the user is at when the load transition happens.

I was running into compatibility issues across mobile devices as it relates to the lines so I removed them entirely for now. I can look into fixing the compatibility issues later on but I was thinking it would be better to get the changes out right now since it still functionally works.

okusio commented:
Perfect! The line looks great. Maybe the shadow could below instead of to the side of the circle?


> On Jan 11, 2021, at 6:39 PM, Chuk116 <notifications@github.com> wrote:
> 
> 
> yessss!!!!
>  <https://user-images.githubusercontent.com/57499412/104249040-a484cb00-5438-11eb-89e9-e814fb3189dc.png>
> Can we make the line skinnier (half the size it currently is). + take away the shadow on the circle. I love love love the checkmark!!!!!
> Also, the line disappears on mobile.
> 
> I can make the line skinnier.
>  <https://user-images.githubusercontent.com/28410610/104250544-a8a5ee00-5422-11eb-9645-6ef6f81e8d80.png>
> The reason I added the shadow is that it serves as an extra subtle indicator for which step the user is at when the load transition happens.
> 
> I was running into compatibility issues across mobile devices as it relates to the lines so I removed them entirely for now. I can look into fixing the compatibility issues later on but I was thinking it would be better to get the changes out right now since it still functionally works.
> 
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub <https://github.com/Representable/representable/pull/575#issuecomment-758293339>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ANWV6FGFCQALQ4ZZTO3RFXDSZOD4DANCNFSM4V6KNSAA>.
>

Chuk116 commented:
F

> Perfect! The line looks great. Maybe the shadow could below instead of to the side of the circle?
> […](#)
> On Jan 11, 2021, at 6:39 PM, Chuk116 ***@***.***> wrote: yessss!!!! <https://user-images.githubusercontent.com/57499412/104249040-a484cb00-5438-11eb-89e9-e814fb3189dc.png> Can we make the line skinnier (half the size it currently is). + take away the shadow on the circle. I love love love the checkmark!!!!! Also, the line disappears on mobile. I can make the line skinnier. <https://user-images.githubusercontent.com/28410610/104250544-a8a5ee00-5422-11eb-9645-6ef6f81e8d80.png> The reason I added the shadow is that it serves as an extra subtle indicator for which step the user is at when the load transition happens. I was running into compatibility issues across mobile devices as it relates to the lines so I removed them entirely for now. I can look into fixing the compatibility issues later on but I was thinking it would be better to get the changes out right now since it still functionally works. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <[#575 (comment)](https://github.com/Representable/representable/pull/575#issuecomment-758293339)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ANWV6FGFCQALQ4ZZTO3RFXDSZOD4DANCNFSM4V6KNSAA>.

Yea definitely!

Chuk116 commented:
> F
> 
> > Perfect! The line looks great. Maybe the shadow could below instead of to the side of the circle?
> > […](#)
> > On Jan 11, 2021, at 6:39 PM, Chuk116 _**@**_.***> wrote: yessss!!!! https://user-images.githubusercontent.com/57499412/104249040-a484cb00-5438-11eb-89e9-e814fb3189dc.png Can we make the line skinnier (half the size it currently is). + take away the shadow on the circle. I love love love the checkmark!!!!! Also, the line disappears on mobile. I can make the line skinnier. https://user-images.githubusercontent.com/28410610/104250544-a8a5ee00-5422-11eb-9645-6ef6f81e8d80.png The reason I added the shadow is that it serves as an extra subtle indicator for which step the user is at when the load transition happens. I was running into compatibility issues across mobile devices as it relates to the lines so I removed them entirely for now. I can look into fixing the compatibility issues later on but I was thinking it would be better to get the changes out right now since it still functionally works. — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub <[#575 (comment)](https://github.com/Representable/representable/pull/575#issuecomment-758293339)>, or unsubscribe https://github.com/notifications/unsubscribe-auth/ANWV6FGFCQALQ4ZZTO3RFXDSZOD4DANCNFSM4V6KNSAA.
> 
> Yea definitely!

Ignore the random F lol

Chuk116 commented:
Here's how the new shadow looks.
<img width="174" alt="Screen Shot 2021-01-12 at 10 32 46 PM" src="https://user-images.githubusercontent.com/28410610/104415088-34547300-5526-11eb-9530-04fa4d5997f2.png">

okusio commented:
Amazing

> On Jan 13, 2021, at 1:33 AM, Chuk116 <notifications@github.com> wrote:
> 
> 
> Here's how the new shadow looks.
>  <https://user-images.githubusercontent.com/28410610/104415088-34547300-5526-11eb-9530-04fa4d5997f2.png>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub <https://github.com/Representable/representable/pull/575#issuecomment-759238329>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ANWV6FEOVGKR7PQLDQDHP73SZU5FLANCNFSM4V6KNSAA>.
>
